src/components/PrimaryPanes/Outline.css

```diff
@@ -2,6 +2,14 @@
   overflow-y: auto;
 }
 
+.outline .outline-pane-info {
+  width: 100%;
+  font-style: italic;
+  text-align: center;
+  padding: 0.5em;
+  user-select: none;
```

Comment:
![image](https://user-images.githubusercontent.com/792924/30354560-2cb124c0-9825-11e7-9ee0-68f052b48f1c.png)

Could we add a `font-size: 12px` so its matches the sizes for `No Breakpoints` and 'Not Paused'.
Atm it looks smaller.